offchain-modules/cli/src/commands/dapp/types.rs

```diff
@@ -46,4 +46,16 @@ pub struct EthTxRelayerArgs {
     pub config_path: String,
     #[clap(long)]
     pub network: Option<String>,
+    #[clap(short = 'p', long)]
+    pub private_key_path: String,
+    #[clap(long, default_value = "100")]
+    pub mint_concurrency: u64,
+    #[clap(long, default_value = "1000")]
+    pub minimum_cell_capacity: u64,
+    #[clap(
+        short = 'd',
+        long,
+        default_value = "mysql://root:@127.0.0.1:3306/forcedb"
```

Comment:
does the default value need to add a password ?

Reply:
you can pass the args with password overwrite default value when start relayer